src/components/_common/Modal/UserModal/UserItems.tsx

```diff
@@ -26,55 +25,66 @@ const UserItems = ({ userId }: { userId: number }) => {
+          <div className="rounded-10 bg-st-gray-50 px-10 py-2 text-12 font-bold text-st-gray-200">
+            {userProfileData.user.position.name}
           </div>
-          <div className="text-18 text-st-gray-400">
+          <div className="flex items-center justify-center gap-10">
+            <div className="text-20 font-bold">
+              {userProfileData.user.nickname}
+            </div>
+            <Link href="/chat">
+              <Icon
+                name={"chat"}
+                size={20}
+                color="text-st-gray-400"
+              />
+            </Link>
+          </div>
+          <div className="text-16 text-st-gray-400">
             {userProfileData.user.bio}
           </div>
         </div>
         <div className="flex w-full flex-col items-center justify-center gap-10">
-          <div className="text-20 font-bold">관심스택</div>
+          <div className="text-15 font-bold">관심스택</div>
           <div>
             {userProfileData.user.stacks.map((stack) => (
-              <div key={stack.id}>{stack.name}</div>
+              <div
+                key={stack.id}
+                className="rounded-10 bg-st-gray-50 px-10 py-2 text-12 font-bold text-st-gray-200"
+              >
+                {stack.name}
+              </div>
             ))}
           </div>
         </div>
         <div className="flex w-full flex-col items-center justify-center gap-10">
-          <div className="text-20 font-bold">받은카드</div>
-          <div className="flex h-80 w-400 flex-col items-center justify-evenly shadow-md">
-            <div className="flex items-center justify-evenly">
-              {userCardsType.map((item, id) => (
-                <div
-                  key={id}
-                  className="text-30"
-                >
-                  {item}
-                </div>
-              ))}
-            </div>
-            <div className="flex">
+          <div className="text-15 font-bold">받은카드</div>
+          <div className="flex h-70 w-400 flex-col items-center justify-evenly shadow-md">
+            <UserCards />
+            <div className="flex w-full items-center justify-evenly">
               {userProfileData.userCards.map((card) => (
                 <div
                   key={card.cardId}
-                  className="text-25"
+                  className="text-18 font-bold"
                 >{`(${card.count})`}</div>
               ))}
             </div>
           </div>
         </div>
-        <div className="flex w-full flex-col items-center justify-center gap-10">
-          <div className="text-15 font-bold">한 줄 평</div>
-          <div>{userProfileData.reviews.slice(0, 3)}</div>
+        <div className="flex w-full flex-grow flex-col items-center justify-center gap-10 overflow-y-auto">
+          <div className="text-13 font-bold">한 줄 평</div>
+          <div className="flex h-200 w-full flex-col overflow-y-scroll">
+            {userProfileData.reviews.map((review, id) => (
```

Comment:
```suggestion
            {userProfileData.reviews.map(({ id, content }) => (
```
review의 내부가 어떻게 생긴지 잘 모르겠지만... 해당 리뷰 데이터의 실제 id값을 key로 넘겨주시는게 좋을 것 같아요!

Reply:
review에는 id값이 따로 없네요 ㅠ